requirements-dev.txt

```diff
@@ -1,5 +1,6 @@
 --index-url https://pypi.devinfra.sentry.io/simple
 
+anton-testing-deleteme-123==3.0.0a1
```

Comment:
this should either be in requirements-base or requirements-dev not both

Reply:
fixed.